### ISSUE TITLE
Provider -> fax numbers?

### ISSUE THREAD

**Issue by eraye1** [ftrotter/vocatus#3]
Is there a list of this that is being worked on?

**Comment by unclenate** [comment]
We have had a couple folks willing to provide what they have from NPI-based data, but I have yet to hear or see anything... Is this soemthign you're willing/able to spearhead?

**Comment by RebeccaCoelius** [comment]
@ftrotter has a good existing mockup using his existing API: http://vocatus.ft1.us/  
Another suggestion he had was https://www.bloomapi.com/

Or Noga of Picnic Health  mentioned we could use her data. They basically started with the NPI list and have been making it better with each records request.

**Comment by untoldone** [comment]
I might consider a non-NPI based approach (or not wholy rely on NPI). While
the NPI is a starting point to bootstrap such a database, you'll find the
quality is pretty low.

If no one has manually collected fax numbers (e.g. Noga + Picnic Health), I
have a number of different datasources in www.bloomapi.com that have fax
numbers and I could give a quick tool that returns the 'most accurate' fax
number from 4 different datasources.

A handful of issues with just the NPI:
- Most providers don't update their listing post med-school, leading to
  a highly outdated fax number
- The fax number for the provider may not be the fax number for records
  release. Providers frequently work at more than one location and the NPI
  only describes the mailing and primary business location of the provider.
- Similarly the records release number for the main system may be
  different than say the radiology records release dept -- even with the same
  provider

**Comment by loranstefani** [comment]
You could use Physician Compare to see some physicians' Group PAC ID, then someone with access to PECOS (not yet open despite standing FOIA request from August 14) could match that to the Group NPI to prioritize higher probability of accurate fax numbers. It has been our experience that groups tend to have more static/better maintained NPPES data than individual physicians.

**Comment by untoldone** [comment]
@loranstefani Do you know anyone that's had luck getting access to PECOS beyond the PECOS Ordering/ Referring data?

You can also use the claims based CCNs + other public data using CCNs in Physician Compare and map that to NPIs through a few different techniques with relatively high matching rates if your goal is to get facility NPIs tied to providers.

There's also pretty sparse but decent fax data in the Provider of Service PUF (CCN keyed) ... but my experience has been this isn't nearly enough as frequently the fax number for a facility is not the same as the fax for records release.

**Comment by loranstefani** [comment]
Nope; my FOIA request for it is still pending and I dont know of anyone else that's requested it. Know who I should bug to get it pushed through? :D

We've matched CCN/NPI from Physician Compare/NPPES for hospital-physician affiliations, but rely on name/address fuzzy matching to build out affiliations between individuals and practices/groups. Not perfect, but may get you a more current fax number than the one listed for an individual. 

Our product (billianshealthdata.com) does verify a large sample of phone numbers for larger groups/practices, but not fax numbers. I see how they're needed for this campaign, but loathe that they are even relevant in 2015 at all.

**Comment by untoldone** [comment]
I've never successfully gotten a FOIA fulfilled faster by pushing on higher up contacts myself (may have at the State level though ;-) )

I've been toying with writing a crawler for fax information -- today I manually lookup fax information/ records release departments to accumulate the data. I would love to connect outside of github -- I've sent you a Linkedin request if you're interested. (sorry for polluting the bug request with this!)
